src/core/atomic.d

```diff
@@ -76,10 +126,10 @@ version( CoreDdoc )
      * Returns:
      *  The result of the operation.
      */
-    HeadUnshared!(T) atomicOp(string op, T, V1)( ref shared T val, V1 mod ) pure nothrow @nogc @safe
+    TailShared!T atomicOp(string op, T, V1)( ref shared T val, V1 mod ) pure nothrow @nogc @safe
```

Comment:
> > Can't do ops on classes, so this is more permissive than before.

> Sorry, I don't follow. What am I allowing that wasn't allowed before?

This function `atomicOp` has nothing to do w/ classes, `TailShared!T` is less restrictive than `HeadUnshared!T` (IIRC), so that function doesn't need any deprecation.